winfsp.go

```diff
@@ -46,10 +49,51 @@ func (mfs *FS) MountWinFSP(mntPoint string) (MountRunner, error) {
 	return s, nil
 }
 
+// foFSImpl is the interface implemented by the
+// the "base" filesystem returned by [gofs.New].
+type goFSImpl interface {
+	winfsp.BehaviourBase
+	winfsp.BehaviourGetSecurityByName
```

Comment:
It's not immediately obvious why we include all these interfaces as well as the base one that is accepted by the `gofs` function signature. AFAIU `gofs.New` optionally switches behaviour on if the type implements these?